src/bin/mosaic-start.ts

```diff
@@ -34,7 +37,30 @@ mosaic
       unlock,
       password,
     });
-
     node.start();
+
+    const nodeRpcPort = rpcPort;
+    if (!options.withoutGraphNode) {
+      const {
+        rpcPort,
```

Comment:
This variable is redundant in the context. Should it be named `graphRPCPort` similarly other ports?